nengo/builder/connection.py

```diff
@@ -76,7 +76,11 @@ def get_targets(conn, eval_points):
     else:
         targets = np.zeros((len(eval_points), conn.size_mid))
         for i, ep in enumerate(eval_points[:, conn.pre_slice]):
-            targets[i] = conn.function(ep)
+            out = conn.function(ep)
+            if out is None:
+                raise BuildError("Building %s: Connection function returned "
```

Comment:
The memory locations go away if the pre and post of the connection have labels. I agree that they're not necessary for most purposes, but they are useful for debugging... might be worth making them something that show up in debug mode only?